src/components/NewsletterSignup/newsletterSignupChangelogData.ts

```diff
@@ -0,0 +1,26 @@
+/** This data is used to populate the ComponentChangelogTable component.
+ *
+ * date: string (when adding new entry during development, set value as "Prerelease")
+ * version: string (when adding new entry during development, set value as "Prerelease")
+ * type: "Bug Fix" | "New Feature" | "Update";
+ * affects: array["Accessibility" | "Documentation" | "Functionality" | "Styles"];
+ * notes: array (will render as a bulleted list, add one array element for each list element)
+ */
+import { ChangelogData } from "../../utils/ComponentChangelogTable";
+
+export const changelogData: ChangelogData[] = [
+  {
+    date: "Prerelease",
+    version: "Prerelease",
+    type: "New Feature",
+    affects: [],
+    notes: ["Adds the NewsletterSignup component to the DS library"],
+  },
+  {
+    date: "Prerelease",
+    version: "Prerelease",
+    type: "New Feature",
+    affects: [],
+    notes: ["Adds the NewsletterSignup component to the DS library"],
+  },
```

Comment:
You only need the first object since the second is redundant.

Reply:
Removed.